src/organisms/Header.jsx

```diff
@@ -20,11 +27,11 @@ const Header = () => {
         <Link to={MAIN_ROUTES.COMPONENT_LIBRARY}>Component Library</Link>
         <div className="site-header-right">
           <Link to={MAIN_ROUTES.HELP}>Help</Link>
-          <span className="usa-media-block__img circle-4 bg-base display-flex flex-row flex-align-center flex-justify-center text-white">
-            {count}
-          </span>
-          {/* @TODO: change user to a link when it exists */}
-          <a href="/user">{userName}</a>
+          {username && (
```

Comment:
Are we not going to have any way for users to edit their information/update password? It seems a little odd that clicking on your username will log you out :/

Reply:
This will be controlled by IDM.  We will be getting this info from CMS central systems and will not be altering it.

Reply:
The logging out thought is a good point and we can bring that up with HCD.

Reply:
I know at some point we plan to get this information from IDM, but currently we are getting it from our own system.

Reply:
We will not being going to production with this login.  This is a temporary measure for piloting.  We will not be editing information that is officially held by CMS identity broker.

Reply:
For the interim, HCD did create designs for how to log out, which involves clicking on the username and then seeing a dropdown menu which you can then click to log out. This work along with any styling or extra features to the sign in page can be done in an additional story if we want to develop this sign in/sign out feature. That story will be able to be prioritized or deprioritized depending on IDM work.